### ISSUE TITLE
blender 3.4.1 animation import from godot missing

### ISSUE THREAD

**Issue by Lightnet** [KhronosGroup/glTF-Blender-IO#1880]
Not sure if post here.

But I found the exporting blender 3.4.1 to godot 4.0 animation is missing when import godot 4.0. It took couple of days to know export for blender 3.4.1 is broken. As learning from video lesson how to create animation and import to godot.

I did test the older version of blender and it worked. I just config the export to match the godot 4.0.

```
# Blender exporter:
 * 2.93.9 animation working
 * 3.3.4 animation working when export and import to godot
 * 3.4.0 animation missing import to godot
 * 3.4.1 animation missing import to godot
```
I create a simple mesh and bone to test the action set and export to file.

I export as testname.glb

**Version**
 - OS: Windows 10 64 bit

**Comment by julienduroure** [comment]
Hello,
As requested in bug template that you deleted, please provide the .blend file you are trying to export

**Comment by Lightnet** [comment]
[testanimation29.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/10962497/testanimation29.zip)

blender-3.3.4
[blendertest03.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/10962516/blendertest03.zip)
Here the sample of the test build. But try to get the latest stable version it does not work.

Here the blender 3.4.1
[testanimation3update.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/10962538/testanimation3update.zip)
Just notice the file size is smaller.

**Comment by julienduroure** [comment]
Hello,
Can you please test with a more recent version of Blender (3.5 beta or 3.6 alpha )
I am not able to reproduce, animations are exported correctly on my side (and imported correctly on Godot 4.0)

**Comment by julienduroure** [comment]
Sorry, but I still can reproduce on blender 3.5 beta
With your 3 files testanimation29, testanimation3update, testanimation3
I just open the file, and export (without changing anything as you checked "remember Export Settings"
--> Animations are here

Because you checked "export selection only", are you sure you don't change the selection?

**Comment by Lightnet** [comment]
I guess it my setting to export. Or incorrect setup but the other version blender 3.6 worked. Only select object for mesh and armture. There default camera and lights. I did not used NLA to export them. Blender 3.4.1. But I try to use NLA and didn't get it working for some reason. All I did simple blank blender project with simple animation test with two animations and still didn't work. Is there reference guide for set up?(although I check the video on export to godot4 that 1 year to 4 moth old, found it not working as shown.)
